DashAI/front/src/App.jsx

```diff
@@ -23,27 +26,45 @@ function App() {
     }
     apiOnlineTest();
   }, []);
-  // const element = useRoutes([
-  //   { path: '/', element: <Home /> },
-  //   { path: '/tabs', element: <ExperimentPipeline /> },
-  // ]);
+
   return (
-    <div>
-      <Navbar />
+    <React.Fragment>
       {apiIsOnline ? (
-        <MainRoutes />
+        <BrowserRouter>
+          <ResponsiveAppBar />
+          <Container maxWidth="lg" sx={{ my: 5, mb: 4 }}>
+            <Grid
+              container
+              direction="row"
+              justifyContent="center"
+              alignItems="center"
+            >
+              <Routes>
+                <Route path="/" element={<Home />} />
+                <Route path="/data/" element={<Data />} />
+                <Route path="/experiments" element={<Experiment />} />
+                <Route path="/results" element={<Results />} />
+                <Route path="/play" element={<Play />} />
```

Comment:
Same as other comment
```suggestion
                <Route path="/app/" element={<Home />} />
                <Route path="/app/data" element={<Data />} />
                <Route path="/app/experiments" element={<Experiment />} />
                <Route path="/app/results" element={<Results />} />
                <Route path="/app/play" element={<Play />} />
```

Reply:
Changed